addons/web_editor/static/image_shapes/convert-to-percentages.js

```diff
@@ -0,0 +1,146 @@
+// marks which axis each parameter of a command belongs to, as well as whether
```

Comment:
This hurt my feelings. I think the indentation is random because I switched from atom to vscode while writing this and my tab-width setting wasn't the same :/ I'd love to turn on an eslint warning for indentation but because the code inside odoo modules is not indented I haven't been able to, maybe with the new modules I could try again. Arthur picked it up half way with the indentation already messed up I guess.

Reply:
> This hurt my feelings.

But the indentation hurt my eyes :cry: Anyways, that was a real question, did not look at the code at all :wink: @detrouxdev Can you reindent the file correctly? Thanks :+1: